contracts/clients/package.json

```diff
@@ -17,15 +17,15 @@
     "publish-experimental-dry-run": "node scripts/showVersion.js >.version && npm version $(node scripts/showBaseVersion.js)-$(git rev-parse HEAD | head -c7) --allow-same-version && npm publish --tag experimental --dry-run && npm version $(cat .version) && rm .version"
   },
   "dependencies": {
-    "ethers": "^5.5.2",
-    "mcl-wasm": "0.7.6",
-    "typescript": "^4.3.5"
```

Comment:
This should only be needed as a dependency to run type checking and transpilation, moved to dev deps.

Reply:
```diff
-This
+typescript
```

👍... for a minute there I was like... why is ethers a dev dependency? 🤯